base_custom_filter/readme/CONFIGURE.rst

```diff
@@ -4,9 +4,12 @@
    * The **Model** for which you are defining the filter.
    * The **Type** depending on whether you want to add a filter or a grouping by
      by a field.
+   * The **Search Field** that will be added to the search.
+   * The **Filter Domain** that will be applied with the search.
    * The **Domain** that will be applied with the filter.
    * The **Group By Field** used to perform the group by.
    * The **Group** to have filters under the same separator.
+   * The **User Groups** that will be applied to the search for the visibility of users.
```

Comment:
Can you re-organize the content by type here? It's now confusing as a type only uses certain set of fields.